client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -58,6 +61,10 @@ export default function LicensePreview( {
 		setOpen( ! isOpen );
 	}, [ isOpen ] );
 
+	const onCopyLicense = () => {
+		dispatch( infoNotice( translate( 'License copied!' ), { duration: 2000 } ) );
+	};
```

Comment:
We can use `useCallback` here to avoid unnecessary renders (e.g. a legit change in state which causes the list item to re-render will cause the details component to re-render as well even if nothing relevant to it changed).